src/main/java/Duke/Task/Events.java

```diff
@@ -0,0 +1,108 @@
+/**
+ * @author Hang Zelin
+ * <p>
+ * Events class that extends Task class. It is one of the types in 3 tasks.
+ * Will contain a time in the form of "/by"
+ */
+package duke.task;
+
+import java.time.LocalDateTime;
+import java.time.format.DateTimeFormatter;
+import java.util.Locale;
+
+public class Events extends Task {
+
+    private boolean done;
+    private String task;
+    private LocalDateTime time;
+    private String taskType = "E";
+
+    /**
+     * Constructor for Events containing boolean value if the task is done, the
+     * specific task info, and the time for the task.
+     *
+     * @param done
+     * @param task
+     * @param time
+     */
+    public Events(boolean done, String task, LocalDateTime time) {
+        this.done = done;
+        this.task = task;
+        this.time = time;
+    }
+
+    /**
+     * Returns the task info in the format of "[type][] task info (/by ...)"
+     *
+     * @return Task info parsed in the format duke executes.
+     */
+    @Override
+    public String getTaskInfo() {
+        String doneStr = "";
```

Comment:
might be good to spell out String. Anyhow since im here maybe doneStatus var name could be considered.